templates/zerver/help/browse-and-join-streams.md

```diff
@@ -49,7 +49,7 @@ subscribe to (highlighted in red).
 
     ![Subscribe before](/static/images/help/subscribe-before.png)
 
-    A green checkmark will appear next to the stream that you've just
+4. A green checkmark will appear next to the stream that you've just
```

Comment:
You put 4. here but on whole file view is 3.

Also please take a look at
{!subscriptions.md!} on whole file view - whats this?